.eslintignore

```diff
@@ -0,0 +1 @@
+./etc/eslint/.eslintignore
```

Comment:
@kgryte The VSCode extension only has an option to pick a working directory relative to which it will resolve `.eslintrc` and `.eslintignore`. Without this, I ended up with a bunch of issues showing up in my editor as I moved to using ESLint for linting both JS and TS files. We already have a top-level `.eslintrc.js` file, too.

Reply:
Okay. Make sense. Would be good to document this in the OP, so we have a future record why certain decisions are made and can reference accordingly.

Reply:
All of that "institutional" knowledge that you have should, ideally, be documented to help prevent future regressions. Including it in the OP is one such place for communicating that knowledge.